VargasMartinezJose/Dia2/HackerRank/ejerciciojava.java/JavaStdinStdout2.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Scanner;
+
+public class JavaStdinStdout2{
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        
+        int ineteger_value = scan.nextInt();
+        double double_value = scan.nextDouble();
+        scan.nextLine();
+        String string_value = scan.nextLine();
```

Comment:
variables in java are written with lowerCamelCase, Don't use _
put a descriptive name